modules/ROOT/pages/provisioning-digitalocean.adoc

```diff
@@ -0,0 +1,49 @@
+= Provisioning Fedora CoreOS on DigitalOcean
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on DigitalOcean.  Fedora CoreOS images are currently not published directly on DigitalOcean, so you must download a Fedora CoreOS DigitalOcean image and upload it to your DigitalOcean account as a https://www.digitalocean.com/docs/images/custom-images/[custom image].
+
+== Prerequisites
+
+Before provisioning a FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to a DigitalOcean account.
+
+== Creating a DigitalOcean custom image
+
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
+
+. Once you have picked the relevant stream, find the corresponding DigitalOcean image on the https://getfedora.org/coreos/download?tab=cloud_operators[download page] and copy the URL of the Download link.
+
+. Create the custom image:
++
+.Example uploading FCOS to a DigitalOcean custom image
+[source, bash]
+----
+doctl compute image create my-fcos-image --region sfo2 --image-url <download-url>
```

Comment:
Unrelated: this looks like another usecase for https://github.com/coreos/coreos-installer/issues/202#issuecomment-606812958.